plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -121,14 +121,17 @@
     private final DockerRegistryAuthResolver authResolver;
     private final ExecutorService            executor;
     private final DockerConnectionFactory    connectionFactory;
+    private final String                     apiVersionPathPrefix;
 
     @Inject
     public DockerConnector(DockerConnectorConfiguration connectorConfiguration,
                            DockerConnectionFactory connectionFactory,
-                           DockerRegistryAuthResolver authResolver) {
+                           DockerRegistryAuthResolver authResolver,
+                           DockerApiVersionPathPrefixProvider dockerApiVersionPathPrefixProvider) {
         this.dockerDaemonUri = connectorConfiguration.getDockerDaemonUri();
         this.connectionFactory = connectionFactory;
         this.authResolver = authResolver;
+        this.apiVersionPathPrefix = dockerApiVersionPathPrefixProvider.get();
```

Comment:
Is it OK to use provider here? Mb it would be better to inject named string here and bind this string to provider?